gradle/findbugs-exclude.xml

```diff
@@ -89,4 +89,54 @@
     <Bug pattern="DMI_RANDOM_USED_ONLY_ONCE, ..." />
   </Match>
 
+  <Match>
```

Comment:
Please don't add findbux exclusions unless the code is taken from somewhere. Even then though we should make an attempt to fix it.